mesh/pmesh.cpp

```diff
@@ -5291,10 +5324,17 @@ long ParMesh::ReduceInt(int value) const
 
 void ParMesh::ParPrint(ostream &out) const
 {
-   if (NURBSext || pncmesh)
+   if (NURBSext)
    {
-      // TODO: AMR meshes, NURBS meshes.
-      Print(out);
+      // TODO: NURBS meshes.
```

Comment:
Shouldn't we give at least a warning in debug mode here?

Reply:
Perhaps... this wasn't my code.